cmd/kubestellar-where-resolver/options/options.go

```diff
@@ -38,19 +38,19 @@ func NewOptions() *Options {
 	logs.Config.Verbosity = config.VerbosityLevel(2)
 
 	return &Options{
-		EspwClientOpts:   *clientoptions.NewClientOpts("espw", "access to the edge service provider workspace"),
-		RootClientOpts:   *clientoptions.NewClientOpts("root", "access to all clusters"),
-		SysAdmClientOpts: *clientoptions.NewClientOpts("sysadm", "access to all clusters as system:admin"),
-		Logs:             logs,
+		EspwClientOpts: *clientoptions.NewClientOpts("espw", "access to the edge service provider workspace"),
+		RootClientOpts: *clientoptions.NewClientOpts("root", "access to the root workspace"),
+		BaseClientOpts: *clientoptions.NewClientOpts("base", "access to the plain apiserver address which is the base of all workspaces"),
```

Comment:
"access" to an "address" is not clear.  The concern is access to API objects in workspaces.  Perhaps a clearer description would be something like the following.

> access to all logical clusters as kcp-admin